crowbar_framework/app/models/service_object.rb

```diff
@@ -1659,11 +1695,33 @@ def active_update(proposal, inst, in_queue)
     end
   end
 
-  def only_if_admin(node)
-    yield if node.admin?
-  end
-
-  def only_unless_admin(node)
-    yield unless node.admin?
```

Comment:
I don't believe you want to remove this.

Reply:
Yes, this seems a mistake, I'll fix it